core/src/test/java/it/pagopa/selfcare/dashboard/core/PartyManagementServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.selfcare.dashboard.core;
+
+import it.pagopa.selfcare.dashboard.connector.rest.client.PartyManagementRestClient;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class PartyManagementServiceImplTest {
+
+    @InjectMocks
+    private PartyManagementServiceImpl partyManagementService;
+
+    @Mock
+    private PartyManagementRestClient restClientMock;
+
+
+    @Test
+    void getOrganization() {
+        // when
+        partyManagementService.getOrganization(null);
+        // then
+        // verifica che il metodo getOrganization del restClientMock sia stato invocato una volta
+        Mockito.verify(restClientMock, Mockito.times(1)).getOrganization(null);
```

Comment:
Consider to verify no other interaction with the mock:
https://www.javadoc.io/doc/org.mockito/mockito-core/2.7.9/org/mockito/Mockito.html#verifyNoMoreInteractions(java.lang.Object...)